### PR TITLE
Devlink (new PR)

### LINKED ISSUE #1: Devlink
Get working with devlink

### PR DESCRIPTION
**Pull request by alekseevaiana**
closes #1 

I already created another PR (https://github.com/UseKeyp/open-dollar-app/pull/5), but Jacob found a way how to run a project from a dev branch, so I created new clean PR. 

- Was able to run a project
- Install Devlink
- Run a project with Test Component from Devlink
- Added instruction to Readme on how to run Devlink

## Issues
- in `useContract.ts` hook file we have to manually ignore typescript rules
- In `webflowrc.js` file I couldn't use variable for `authToken` from `.env.development.local` file. So I added the whole `webflowrc.js` file to `.gitignore`

## Screenshot
Screenshot 2023-07-17 at 2 32 57 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,25 @@ To have the app default to a different network when a wallet is not connected:
 2. Change `REACT_APP_NETWORK_ID` to `"420"`
 3. Change `REACT_APP_NETWORK_URL` to e.g. `"https://opt-goerli.g.alchemy.com/v2/{YOUR_INFURA_KEY}"`
 
+### Configuring the devlink
```
Comment (pi0neerpat) [inline README.md:41]: Awesome thank you!

File: package.json [hunk 2]
```diff
@@ -41,6 +41,7 @@
         "@web3-react/injected-connector": "^6.0.7",
         "@web3-react/walletconnect-connector": "6.2.4",
         "@web3-react/walletlink-connector": "6.2.3",
+        "@webflow/webflow-cli": "^1.5.1",
```
Comment (pi0neerpat) [inline package.json:44]: Should this be a dev dependency?

**Review by pi0neerpat — APPROVED** [review]
Do we need all these `_Builtin` components? If not, please remove them

**Comment by alekseevaiana** [comment]
> Do we need all these `_Builtin` components? If not, please remove them

I don't think we can edit devlink folder, because every time we sync it with a webflow and it gets all files from there

File: package.json (hunk 2 above)
Comment (alekseevaiana) [reply package.json:44]: Couldn't find a reference on this in documentation, but the command I used to install it, installed it in a dependencies. But I can change it manually I guess.

**Pull request merged**